cmake/common/helpers_common.cmake

```diff
@@ -499,8 +499,18 @@ function(add_obs_plugin target)
     set(found_architecture TRUE)
   else()
     foreach(architecture IN LISTS _AOP_ARCHITECTURES)
-      if("${architecture}" STREQUAL CMAKE_SYSTEM_PROCESSOR)
-        set(found_architecture TRUE)
+      if(OS_WINDOWS AND CMAKE_GENERATOR MATCHES "Visual Studio .+")
```

Comment:
We usually don't explicitly check for Visual Studio (because we don't support any other way to build OBS on Windows). It's fine as-is, but it's implicitly expected and stuff is expected to break if you don't use MSVC.

Reply:
I remove the check, if only MSVC is supported as generator.